spec/index.adoc

```diff
@@ -1797,7 +1797,7 @@ For a newly discovered block, a regular Bitcoin (full) node therefore provides a
 than the Bitcoin component, which implies that it is advisable to set the number of confirmations to a reasonably large value, such as 6,
 to gain confidence in the correctness of the returned UTXOs.
 
-There is an upper bound of 100,000 UTXOs that can be returned in a single request. For addresses that contain sufficiently many UTXOs,
+There is an upper bound of 1,000 UTXOs that can be returned in a single request. For addresses that contain sufficiently many UTXOs,
```

Comment:
I actually think we should put this higher at, say, 10k. The implementation can still cap it at 1,000, but having the upper bound here by 10k allows us to increase the implementation's cap without breaking the spec.

Reply:
Sounds good to me, @THLO, are you fine with 10K as well?

Reply:
Yes, that's the number @ielashi and I agreed on.

Reply:
Set to 10,000 as agreed.